### ISSUE TITLE
Exception: <type 'exceptions.OSError'>

### ISSUE THREAD

**Issue by freedreamer82** [lukasjapan/bt-speaker#50]
Hi , I tried to start the daemon manually and i get thi stuff ( raspberry 3 ).
I see the device on my phone and it tries to connect to it but suddenly the connection drops..

any idea ?

thanks

Marco

Device connected. device=/org/bluez/hci0/dev_B4_9C_DF_3C_D9_26
Hiding adapter from all devices.
+ ogg123 /usr/share/sounds/freedesktop/stereo/service-login.oga

Audio Device:   Advanced Linux Sound Architecture (ALSA) output

Playing: /usr/share/sounds/freedesktop/stereo/service-login.oga
Ogg Vorbis stream: 2 channel, 22050 Hz

Done.
Exception: <type 'exceptions.OSError'>
Device disconnected. device=/org/bluez/hci0/dev_B4_9C_DF_3C_D9_26
Showing adapter to all devices.
+ o

**Comment by xfjx** [comment]
I am getting the same error after reinstalling.

Did you find out the root of the problem? Maybe it is related to an updated bluetooth package in rasbian?

**Comment by xfjx** [comment]
I think my problem was ralated to another app already having the alsa device open.

**Comment by freedreamer82** [comment]
yes could be. I had another app using alsa as well, stopping that , now it works.

**Comment by schlimmchen** [comment]
> yes could be. I had another app using alsa as well, stopping that , now it works.

@freedreamer82 In that case you might want to close this issue, please.
